packages/deployments/contracts/contracts/core/connext/facets/BridgeFacet.sol

```diff
@@ -80,8 +74,9 @@ contract BridgeFacet is BaseConnextFacet {
   event XCalled(
     bytes32 indexed transferId,
     uint256 indexed nonce,
+    uint256 bridgedAmt,
     XCallArgs xcallArgs,
-    XCalledEventArgs args,
+    address bridged,
```

Comment:
can we reorder to have bridged and bridgedAmt next to each other? also can we rename to bridgedAmount and bridgedAsset?

Reply:
or maybe bridgedAssetId to match args.transactingAssetId